test/specs/lwcLsp.e2e.ts

```diff
@@ -0,0 +1,124 @@
+      const text = await item.getText();
+      if (text.includes('Install and Reload')) {
+        await item.click();
+      }
+    }
+    await utilities.pause(10);
+    buttons = await $$('a.monaco-button.monaco-text-button');
+    for (const item of buttons) {
+      const text = await item.getText();
+      if (text.includes('Reload and Enable Extensions')) {
+        await item.click();
+      }
+    }
+    await utilities.pause(5);
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'Developer: Reload Window',
+      10
+    );
+
+    // Verify Lightning Web Components extension is present and running
+    const extensionNameDivs = await $$('div.name');
+    let extensionWasFound = false;
+    for (const extensionNameDiv of extensionNameDivs) {
+      const text = await extensionNameDiv.getText();
+      if (text.includes('salesforce.salesforcedx-vscode-lwc')) {
+        extensionWasFound = true;
+      }
+    }
+    expect(extensionWasFound).toBe(true);
+  });
+
+  step('Go to Definition (Javascript)', async () => {
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    const textEditor = (await editorView.openEditor('lwc1.js')) as TextEditor;
+    await textEditor.moveCursor(3, 40);
+
+    // Go to definition through F12
+    await browser.keys(['F12']);
+    await utilities.pause(1);
+
+    //Verify 'Go to definition' took us to the definition file
+    const activeTab = await editorView.getActiveTab();
+    const title = await activeTab?.getTitle();
+    expect(title).toBe('engine.d.ts');
+  });
+
+  step('Go to Definition (HTML)', async () => {
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    const textEditor = (await editorView.openEditor('lwc1.html')) as TextEditor;
+    await textEditor.moveCursor(3, 57);
+
+    // Go to definition through F12
+    await browser.keys(['F12']);
+    await utilities.pause(1);
+
+    //Verify 'Go to definition' took us to the definition file
+    const activeTab = await editorView.getActiveTab();
+    const title = await activeTab?.getTitle();
+    expect(title).toBe('lwc1.js');
+  });
+
+  step('Autocompletion', async () => {
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    const textEditor = (await editorView.openEditor('lwc1.html')) as TextEditor;
+    await textEditor.typeTextAt(3, 11, ' lwc');
+    await utilities.pause(1);
+
+    // Verify autocompletion options are present
+    const autocompletionOptions = await $$(
+      'textarea.inputarea.monaco-mouse-cursor-text'
+    );
+    expect(await autocompletionOptions[0].getAttribute('aria-haspopup')).toBe(
+      'true'
+    );
+    expect(
+      await autocompletionOptions[0].getAttribute('aria-autocomplete')
+    ).toBe('list');
```

Comment:
Would it also be possible to select any of the options and see that it gets inserted into the document?